src/About/DeviceCard.tsx

```diff
@@ -49,12 +49,13 @@ export default () => {
             <Section title="ID">{device.serialNumber}</Section>
             <Section title="PCA">{pca || 'Unknown'}</Section>
             <Section title="Cores">{cores || 'Unknown'}</Section>
+            {/* 
```

Comment:
This is an existing bug, doesnt work today. Commented out and made a trello https://trello.com/c/9XBs3jZF/169-fix-device-info-in-about-pane